lib/git/branch_shortcuts.sh

```
@@ -49,25 +49,25 @@ function _scmb_git_checkout_shortcuts {
     return $?
   fi
 
+  # If only branch then check if it's a worktree
   if [[ "$1" =~ ^[0-9]+$ ]]; then
     local branch_var="${git_env_char}$1"
```

Shellcheck (0.11.0)

[warning] git_env_char is referenced but not assigned.

(SC2154)